### PR TITLE
Improved supermatter headaches and voice

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: You only have a headache looking at the supermatter if you're a human without mesons.
add: The supermatter now speaks in a robotic fashion.
/:cl:

- Ghosts can't get headaches.
- Look, I know you're trying really hard as the syndicate comms agent, but maybe you should just try doing something else.

### CODE REVIEW

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -90,6 +90,22 @@
 		countdown = null
 	. = ..()
 
+/obj/machinery/power/supermatter_shard/examine(mob/user)
+	..()
+	if(!ishuman(user))
+		return
+
+	var/range = min(7, round(power ** 0.25))
```
Comment (ChangelingRain) [inline code/modules/power/supermatter/supermatter.dm]: this formula should probably be a proc or something so you can use it in both places it's used

**Comment by kevinz000** [comment]
fun removal

File: code/modules/power/supermatter/supermatter.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/power/supermatter/supermatter.dm]: please make it a macro and replace the other location as well

Thanks

**Comment by coiax** [comment]
@optimumtact done

**Pull request merged**